libpod/image.go

```diff
@@ -55,11 +70,12 @@ type ImageFilter func(*storage.Image) bool
 // pulled. If allTags is true, all tags for the requested image will be pulled.
 // Signature validation will be performed if the Runtime has been appropriately
 // configured
-func (r *Runtime) PullImage(imgName string, allTags bool, reportWriter io.Writer) error {
+func (r *Runtime) PullImage(imgName string, allTags bool, options CopyOptions, reportWriter io.Writer) error {
```

Comment:
I don't like having a full CopyOptions here. Don't we just need the signature policy path? Can we pass that in as a string instead of passing an entire CopyOptions?

Reply:
yeah we can. I was trying to keep it symmetrical with `PushImage`.

Reply:
Also, can get get locking and a validity check on the runtime here? Don't want to pull images when c/storage is shut down.

Reply:
I agree drop the CopyOptions.

Reply:
fixed